contracts/atomic-order-example/Cargo.toml

```diff
@@ -29,15 +27,15 @@ optimize = """docker run --rm -v "$(pwd)":/code \
 """
 
 [dependencies]
-cosmwasm-std       = { version = "1.5.0", features = [ "abort", "cosmwasm_1_2", "cosmwasm_1_3", "cosmwasm_1_4", "iterator", "stargate" ] }
-cosmwasm-storage   = { version = "1.5.0", features = [ "iterator" ] }
-cw-storage-plus    = { version = "1.2.0" }
-cw-utils           = { version = "0.16.0" }
-cw2                = { version = "0.16.0" }
-injective-cosmwasm = { version = "0.2.23-rc1", path = "../../packages/injective-cosmwasm" }
-injective-math     = { version = "0.2.4", path = "../../packages/injective-math" }
-injective-protobuf = { version = "0.2.2", path = "../../packages/injective-protobuf" }
-protobuf           = { version = "2.28.0", features = [ "with-bytes" ] }
+cosmwasm-std       = { version = "2.1.0", features = [ "abort", "cosmwasm_1_2", "cosmwasm_1_3", "cosmwasm_1_4", "cosmwasm_2_0", "iterator", "stargate" ] }
+cosmwasm-storage   = { version = "1.5.2", features = [ "iterator" ] }
+cw-storage-plus    = { version = "2.0.0" }
+cw-utils           = { version = "2.0.0" }
+cw2                = { version = "2.0.0" }
+injective-cosmwasm = { version = "0.3.0" }
+injective-math     = { version = "0.3.0" }
+injective-std      = { version = "1.13.0" }
```

Comment:
as discussed previously, please change these dependencies (and everywhere else) to root level Cargo.toml instead. and please also keep the relative path, see

https://www.reddit.com/r/rust/comments/i0atzm/how_should_you_handle_publishing_crates_that/

and from the [docs](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#multiple-locations):

> It is possible to specify both a registry version and a git or path location. The git or path dependency will be used locally (in which case the version is checked against the local copy), and when published to a registry like crates.io, it will use the registry version. Other combinations are not allowed.